src/main/controllers/UserManagementController.ts

```diff
@@ -14,12 +15,14 @@ export default class UserManagementController {
             res.redirect(`user-management?${filterValues}`);
         } else {
             const pageData = await userManagementService.getFormattedData(
-                parseInt(<string>req.query.page) || 1,
-                (req.query?.email as string) || '',
-                (req.query?.userId as string) || '',
-                (req.query?.userProvenanceId as string) || '',
-                (req.query?.roles as string) || '',
-                (req.query?.provenances as string) || '',
+                new UserSearchCriteria(
+                    req.query.page,
```

Comment:
"req.query" is not mark as optional here but they are flagged as optional in the subsequent parameters. Can make them consistent

Reply:
Good spot, now marked as optional :)